apps/nextjs/src/app/api/chat/chatHandler.ts

```diff
@@ -180,9 +193,12 @@ export async function handleChatPostRequest(
               messages,
             },
             services: {
-              chatLlmService: llmService,
+              chatLlmService: () =>
+                llmService ?? new OpenAIService({ userId, chatId }),
               moderationAiClient,
               ragService: (aila: AilaServices) => new AilaRag({ aila }),
+              chatCategoriser: (aila: AilaServices) =>
```

Comment:
Pass in the categoriser at initialisation, in the same way we do with the other features